cmd/tests/cmd_test.go

```diff
@@ -22,160 +22,186 @@ import (
 	"github.com/stretchr/testify/require"
 	"github.com/tidwall/gjson"
 	"go.k6.io/k6/cloudapi"
+	"go.k6.io/k6/cmd"
 	"go.k6.io/k6/errext/exitcodes"
 	"go.k6.io/k6/lib"
 	"go.k6.io/k6/lib/consts"
 	"go.k6.io/k6/lib/testutils"
 	"go.k6.io/k6/lib/testutils/httpmultibin"
 )
 
+func TestDeprecatedOptionWarning(t *testing.T) {
```

Comment:
This test also seems duplicated in `cmd/root_test.go`, in `TestDeprecatedOptionWarning`. I don't have a strong opinion where it should be, but it shouldn't be in both places. And given that it _can_ be in `cmd/` and was there originally, my vote would be to delete this from here.

Reply:
:facepalm: I thought I removed all the duplication. Sorry, bad conflict resolution, will fix.

Reply:
Fixed in 7216a745.